src/api/article/localized/controller.js

```diff
@@ -6,6 +6,7 @@ import Article from 'api/article/article.model';
 
 import LocalizedArticle from './model';
 
+// TODO(uladbohdan): to refactor using then-chaining.
```

Comment:
why?

Reply:
I actually thought that is the plan for such cases.